src/editors/containers/VideoUploadEditor/index.scss

```diff
@@ -0,0 +1,64 @@
+.dropzone-wrapper {
```

Comment:
@pkulkark IIRC, the convention is to use `paragon` and `Bootstrap` classes as much as possible (within reason) and try to reduce the SCSS.

Reply:
@Cup0fCoffee Done. I moved most of it out of the SCSS file. Only kept a few custom styling.